src/components/Nav/_nav-item.scss

```diff
@@ -112,6 +83,35 @@
       align-items: stretch;
     }
   }
+
+  @include breakpoint(sm) {
+    padding-left: calc(4 * var(--space-unit));
+    padding-right: calc(4 * var(--space-unit));
+  }
+
+  @include breakpoint(md) {
+    color: var(--colour-white);
+    padding: calc(2 * var(--space-unit));
+
+    &:active,
+    &:focus,
+    &:hover,
+    &.active {
+      color: var(--colour-white);
+    }
+
+    .nav__link-text:before {
```

Comment:
personally would write these with media queries inside the nested selector, to avoid repeated selectors, e.g.

```css
.top-level-selector {
  color: blue;

  @media (min-width: 12px) {
    color: black;
  }
 
  .nested-selector {
    color: blue;
    
    @media (min-width: 12px) {
      color: black;
    }

    @media (min-width: 24px) {
      color: red;
    }
  }
}
```